### DISCUSSION TITLE
Persist glance on tab switch

### DISCUSSION THREAD

**Discussion by Marin-Kitagawa** [zen-browser/desktop#2718] [Ideas]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

Now, the glance window just vanishes when I switch to a new tab. Please make it at least like Arc browser's peek. It will persist regardless of the tab switch. Also, I want to be able to do anything inside the glance such as go inside a link in the glance window, go back and forward.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.17 (Firefox 132.0) (64-bit)

### What platform are you seeing the problem on?

Linux, Windows

**Comment by Duffed** [comment]
Just want to add here that having it disappear on a tab switch makes using glance useless in a lot of cases for me. So +1

**Comment by 12th-devs** [comment]
+1
